README.md

```diff
@@ -14,7 +14,8 @@ different PostgreSQL server versions without introducing server version
 check code blocks.  Please choose a branch to match the PostgreSQL version
 you will be building pg_rman against.
 
-* master : branch for PostgreSQL 15 [![Test](https://github.com/ossc-db/pg_rman/actions/workflows/build.yml/badge.svg?branch=master&event=push)](https://github.com/ossc-db/pg_rman/actions/workflows/build.yml)
+* master : branch for PostgreSQL 17 [![Test](https://github.com/ossc-db/pg_rman/actions/workflows/build.yml/badge.svg?branch=master&event=push)](https://github.com/ossc-db/pg_rman/actions/workflows/build.yml)
+* REL_16_STABLE : branch for PostgreSQL 16 [![Test](https://github.com/ossc-db/pg_rman/actions/workflows/build.yml/badge.svg?branch=REL_15_STABLE&event=push)](https://github.com/ossc-db/pg_rman/actions/workflows/build.yml)
```

Comment:
At present, REL_16_STABLE does not exist, so leave it at 15 on the link below.

Reply:
I think we will forget to change it. Since you will make the branch soon, I think it's better to change for v16